### ISSUE TITLE
Icons have wrong name

### ISSUE THREAD

**Issue by Luis-Henriquez-Perez** [domtronn/all-the-icons.el#99]
I have the icons installed but for some reason they are named incorrectly. 
For example after checking the icons and their respective names you can see pretty much all of them don't correspond to the right icon. 
![bad-icons](https://user-images.githubusercontent.com/30783193/35674608-2dd1424c-0713-11e8-9e50-e44c3b64003e.png)

**Comment by md1023** [comment]
I've evaluated `(insert (all-the-icons-octicon "file-directory" :v-adjust all-the-icons-dired-v-adjust :face 'all-the-icons-dired-dir-face))` in scratch buffer and got lighting symbol instead of folder. See details in https://github.com/jtbm37/all-the-icons-dired/issues/13.

Something wrong happened since I've upgraded to `20180125.757` version. Strangely dired icons broke, but neotree looks as it should.

Advise how to debug issue.

**Comment by md1023** [comment]
I disabled font-lock `M-x global-font-lock-mode RET`, refreshed dired buffer and all icons got proper glyphs!

**Comment by leodumont** [comment]
@md1023 Thanks, it works for me too.

**Comment by Luis-Henriquez-Perez** [comment]
For me when I disable the font lock and then refresh the dired buffer the icons all become folders. I don't think this is completely right as I recall when the icons were the right images, some folders (like the download folder) was different. 
Also for me this doesn't last. After the font lock and refresh when I exit dired and return to dired, the icons are wrong again.

**Comment by md1023** [comment]
My solution is temporary, I put a hook:
```
(defun simno-dired-mode-setup ()
  "show less information in dired buffers"
  (dired-hide-details-mode 1)
  (local-set-key (kbd "TAB") 'dired-subtree-cycle)
  (font-lock-mode 0)
)

(add-hook 'dired-mode-hook 'simno-dired-mode-setup)
```
Looking forward to new versions. Some folks don't have problems on Emacs 26, we'll see.

**Comment by andersjohansson** [comment]
See #105 for more info and a proposed fix in #106

**Comment by Luis-Henriquez-Perez** [comment]
This works. However I will note that I needed to manually replace the functions added with `font-lock+` using overriding advice. I don't mind this because I think it's better practice to override using advising than to overwrite it by defining a function with the same name. Since the code of `font-lock+` is pretty short I just copied over the functions to my config and appended a '+' to them so that they can be distinguished from the original.

For example:
;;   override original with new
`(advice-add 'font-lock-default-unfontify-region :override #'+font-lock-default-unfontify-region)`

**Comment by seagle0128** [comment]
This issue still exists with GNU Emacs 25 on Ubuntu 18.04..

**Comment by ZzAntares** [comment]
Same issue here, what's exactly the fix? to install `font-lock+.el`? to manually edit `all-the-icons.el`?

**Comment by uKLEina** [comment]
I had the same problem.

I use `el-get.el` to install `all-the-icons.el` and it requires `font-lock+.el` in its recipe, so I already had `font-lock+.el` itself.

However, it seems `all-the-icons.el` doesn't load  `font-lock+.el` in it, so just add
`(require 'font-lock+)`
in the `.emacs` and that solves the problem for me, i.e. shows icons correctly and font-lock is still enabled on the other characters.

**Comment by buzztaiki** [comment]
I send PR https://github.com/jtbm37/all-the-icons-dired/pull/21 to fix this issue of all-the-icons-dired.

**Comment by wyuenho** [comment]
Fixed by #106
